app/partials/errorDetail.html

```diff
@@ -14,8 +14,8 @@
     </thead>
     <tbody ng-repeat="item in error.errors | paginate:start():end()">
         <tr ng-repeat="(key, value) in item.properties">
-            <th class="id" scope="row" rowspan="{{item.properties | keyLength}}" ng-if="$first">{{item.loanNumber || item.lineNumber}}</th>
-            <td class="description">{{key | hmdaLabel:error.scope}}</td>
+            <td class="id" scope="row" rowspan="{{item.properties | keyLength}}" ng-if="$first">{{item.loanNumber || item.lineNumber}}</td>
```

Comment:
Not sure how that got changed.. yes, it should be `th`. Must have been something Sublime did for me and I didn't notice. Good catch.